integr8ly/integreatly-operator
# INTLY-6744 - Role for RHMI Config

# Description

Dedicated admins should have view and update access to RHMI Config resource

Jira:
* https://issues.redhat.com/browse/INTLY-6744

## Verification
(Already covered as part of e2e test)
* Install RHMI from this branch 
  * Should only need to wait till the `bootstrap` phase is complete
* Verify `rhmiconfig-dedicated-admins-role` Role is created
* Verify `rhmiconfig-dedicated-admins-role-binding` Role Binding is created
* Install testing IDP
  ```
  PASSWORD=Password1 ./scripts/setup-sso-idp.sh
  ```
* Login as a dedicated admin (e.g `customer-admin01`)
* Verify dedicated admin **can** view and update RHMI Config in only the `redhat-rhmi-operator` namespace
* Verify dedicated admin **cannot** delete or create RHMI config resource
* Login as developer user (e.g `test-user01`)
* Verify test user has no access to the RHMI Config resource at all

## Type of change




## Checklist
